### PR TITLE
user keywords

### PR DESCRIPTION
**Pull request by yingjieMiao**
@eishay

### CODE REVIEW

**Comment by eishay** [comment]
Thanks @yingjieMiao 
May I ask that @stkem (or someone else if there's a taker) would do the review buddy for the cortex feature side code? I'm simply not in capacity to do it this week.

**Comment by stkem** [comment]
I'll review it. In the middle of something right now (http://peterhrynkow.com/wp-content/uploads/2012/10/post-18529-Yes-mlkshk-1nAi.gif), but will review within the hour.

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminBookmarkController.scala [hunk 1]
```diff
@@ -218,5 +218,28 @@ class AdminBookmarksController @Inject() (
       Ok(html.admin.bookmarks(bookmarksAndUsers, page, overallCount, pageCount, keeperKeepCount, privateKeeperKeepCount, tweakedCounts, total))
     }
   }
+
+  def userBookmarkKeywords = AdminHtmlAction.authenticatedAsync { request =>
+    val user = request.userId
+    val uris = db.readOnly{ implicit s =>
+      keepRepo.getByUser(user)
+    }.filter(!_.isPrivate).map{_.uriId}.sortBy( x => x.id)    // sorting helps s3 performance
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminBookmarkController.scala:226]: Interesting! Is that documented or did you observe that in prod?
Also, if it's purely for performance reasons, might be better in the Commander.

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminBookmarkController.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminBookmarkController.scala:226]: saw it here
`http://aws.amazon.com/articles/1904`

didn't really do any benchmark.

**Pull request merged**